homeassistant/components/sensor/recorder.py

```diff
@@ -0,0 +1,92 @@
+"""Statistics helper for sensor."""
+from __future__ import annotations
+
+import datetime
+import logging
+import statistics
+
+from homeassistant.components import history
+from homeassistant.const import ATTR_DEVICE_CLASS
+from homeassistant.core import HomeAssistant
+
+from . import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+DEVICE_CLASSES = {"temperature"}
```

Comment:
Do you need this if you can just check/iterate over the keys of `DEVICE_CLASS_STATISTICS` ?

Reply:
Fixed in 407dc9d